nansat/vrt.py

```diff
@@ -1064,8 +1064,6 @@ def create_band(self, src, dst=None):
             srcs = [src]
         elif type(src) in [list, tuple]:
             srcs = src
-        else:
-            raise ValueError('Wrong src type (%s)! Should be dict or list/tuple of dict'%type(src))
```

Comment:
Why did you remove the Exception? It is potentially possible to get a wrong data type

Reply:
It is potentially possible to give wrong input in any method. To keep it simple I suggest not to add checks of inputs without strict necessity.